playbooks/kube-install.yml

```diff
@@ -13,24 +13,30 @@
   become: true
   become_user: root
   tasks: []
+  pre_tasks:
+    - set_fact:
+        calc_proxy_env: "{{ proxy_env|calculate_no_proxy }}"
+    - group_by: key={{ ansible_os_family }}
   roles:
+    - { role: set-proxy, when: calc_proxy_env }
     - { role: bridge-setup,
         when: pod_network_type == 'bridge' or pod_network_type == 'kokonet-bridge' }
     - { role: kokonet-setup, when: pod_network_type == 'kokonet-bridge' }
     - { role: npwg-poc1-setup, when: pod_network_type == 'multus' and multus_npwg_demo }
-    - { role: optional-packages }
-    - { role: install-go, when: container_runtime == 'crio' }
+    - { role: optional-packages, optional_packages: "@Development tools" }
```

Comment:
move to new extra-vars file, you can specify the `optional_packages` there with the `@Development tools` group.